### PR TITLE
# Add es6 syntax for the project

### PR DESCRIPTION
**Pull request by mwafrika**
# The following ES6 syntax has been added to the project:

- Arrow functions 
- const and let 
- Object destructuring
- removing redundant values in the object (E.g: const b = {a:a, b:b}  to be const b ={a, b})

[LIVE DEMO](https://mwafrika.github.io/awasome-book-es6/)

### Thanks for your time 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,12 @@
 
 # Awesome Books Project
 
-> A web app to add your books in a digital shelf. This webapp is built with vanila javascript. A local storage option is added with this project, so that a user can store data in this is web app. 
+> A web app to add your books in a digital shelf. This webapp is built with vanila javascript. A local storage option is added with this project, so that a user can store data in this is web app.
```
Comment (rustam-tolipov) [inline README.md:5]: - [ ] The screenshots are not showing in the readme file. Please kindly fix this issue 📸

File: modules/js/index.js [hunk 2]
```diff
@@ -3,7 +3,12 @@ const inputAuthor = document.querySelector('.input-author');
 const bookShelf = document.querySelector('.book-shelf');
 const bookCard = document.createElement('div');
 const bookForm = document.querySelector('.book-input');
-
+const navlist = document.querySelector('#show-list-button');
+const navadd = document.querySelector('#add-new-button');
+const navcon = document.querySelector('#contact-button');
+const list = document.querySelector('#list');
+const addNew = document.querySelector('#add');
+const connav = document.querySelector('#contact');
```
Comment (rustam-tolipov) [inline modules/js/index.js]: ## GOOD JOB your app is working well already 👍

- [ ] Let's divide the js files into reasonable modules and put them into the `modules` folder. And also `css` and `images` shouldn't be inside this folder 💡

File: modules/js/index.js [hunk 3]
```diff
@@ -94,45 +100,36 @@ bookForm.addEventListener('submit', (e) => {
...
 navlist.addEventListener('click', listfun);
 
 navadd.addEventListener('click', addfun);
 
 navcon.addEventListener('click', confun);
 
-// eslint-disable-next-line no-unused-vars
 const displayTime = () => {
   document.getElementById('current-date').innerText = new Date().toLocaleString();
   setTimeout(displayTime, 1000);
```
Comment (rustam-tolipov) [inline modules/js/index.js]: - [ ] Great job with implementing the display date function 🙌. However, you should use the [Luxon](https://moment.github.io/luxon/#/install) library to display the time ⌚. Please kindly install this package and use it to display the time, for more information use this article: [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mwafrika) [reply README.md:5]: Alright, well noted. I am going to update that

File: modules/js/index.js (hunk 2 above)
Comment (mwafrika) [reply modules/js/index.js]: Yes. I will work on it soon

File: modules/js/index.js (hunk 3 above)
Comment (mwafrika) [reply modules/js/index.js]: Thanks for the information. I am going to use that now

File: modules/index.js [hunk 4]
```diff
@@ -94,47 +104,40 @@ bookForm.addEventListener('submit', (e) => {
...
 navadd.addEventListener('click', addfun);
 
 navcon.addEventListener('click', confun);
 
-// eslint-disable-next-line no-unused-vars
 const displayTime = () => {
-  document.getElementById('current-date').innerText = new Date().toLocaleString();
+  document.getElementById('current-date').innerText = DateTime.now().toFormat(
+    "HH 'hours and' mm 'minutes'",
+  );
```
Comment (omar-labana) [inline modules/index.js]: - [ ] According to the project requirements, the luxon library should be used to display the current `date and time` and what is displayed is only the time in 24hr format, please update it to show the current date and time.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @mwafrika, 

Excellent work so far! 👌
You have implemented all the required features correctly without any bugs or errors. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/index.js (hunk 4 above)
Comment (mwafrika) [reply modules/index.js]: Everything is alright now. you can check the update on the readme now. and in the index.js file

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @mwafrika,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**